### DISCUSSION TITLE
QChatGPT 3.0 发行说明和迁移指南

### DISCUSSION THREAD

**Discussion by RockChinQ** [langbot-app/LangBot#690] [Show and tell]
3.0 中进行了大规模的重构，保留了大部分原有的功能，删除了一些使用率很低的功能，提高了程序整体上的扩展性和可维护性，以下是详细更改。  
官方文档已更新：https://qchatgpt.rockchin.top  
详细更改见：https://github.com/RockChinQ/QChatGPT/pull/669

旧版 2.x 的文档保留在： https://v2.qchatgpt.rockchin.top

**Comment by RockChinQ** [comment]
占位

**Comment by RockChinQ** [comment]
占位

**Comment by SQSora** [comment]
3.0从配置文件到系统架构都进行了大调整，官方Wiki中为何不考虑保留v2版本的文档呢

**Reply by RockChinQ** [reply]
有： https://v2.qchatgpt.rockchin.top
只是目前还没在文档上贴出。
